### PR TITLE
Various PNI styling fixes

### LINKED ISSUE #3752: PNI: implement updated homepage banner
Update banner header image with this new graphic [here](https://drive.google.com/drive/folders/1CeZ3nTJ3zIyqRSqUtj7r4cjOYvktq3qk?usp=sharing).

### LINKED ISSUE #3767: PNI: update badge
- [ ] Update large badge on product detail page [@natalieworth to add svg]

- [ ] Update mini badge on homepage [@natalieworth to add svg]

### LINKED ISSUE #3791: PNI icons sometimes overlap
Noticed that if a product is 18+ and filters are being used then the icons overlap. @beccaklam could you please find an alternative spot for the one of the overlapping icons?

Screen Shot 2019-09-04 at 12 22 28 PM

### LINKED ISSUE #3795: PNI: differences between picture background and css background
**Describe the bug**
There is a small difference of color between the css and the image background on the PNI page, which results in a faint square around the image:
image

URL here: https://foundation.mozilla.org/en/privacynotincluded/

**Expected behavior**
To have the same color.

**Additional context**
Changing the class ".bg-gray" to #f5f5f5 solves the issue but I am not sure about how this affects other pages.
image

### PR DESCRIPTION
**Pull request by mmmavis**
will assign code reviewer after design R+

Closes #3795 
Closes #3767 
Closes #3752 
Closes #3791 

Page: https://foundation-mofostaging-pr-3802.herokuapp.com/en/privacynotincluded/products/interview-deep/
Edit: https://foundation-mofostaging-pr-3802.herokuapp.com/cms/buyersguide/product/edit/8/

### CODE REVIEW

File: source/sass/_colors.scss [hunk 1]
```diff
@@ -28,7 +28,8 @@ $pni-purple-pink: #b0379b;
 $pni-pink: #e4487d;
 $pni-gradient: linear-gradient(to right, $pni-blue 0%, $pni-pink 100%);
 
-$bg-product-image: #f4f4f4; /* keep this #f4f4f4 as this is the background colour that product images have */
+$pni-bg-product-list: #f5f5f5; /* keep this #f5f5f5 as this is the background colour product images on Homepage have */
```
Comment (Pomax) [inline source/sass/_colors.scss]: the `bg` is "buyersguide", so we probably want to replace `bg` with `pni`, rather than adding it at the front?

Comment (mmmavis) [reply source/sass/_colors.scss]: Ah I used `bg` for `background`. Would replacing `bg` here with `background` help?

Comment (mmmavis) [reply source/sass/_colors.scss]: Ah I used `bg` for `background`. I will replace `bg` here with `background` to make it more explicit.

Comment (mmmavis) [reply source/sass/_colors.scss]: Updated!

**Comment by mmmavis** [comment]
@Pomax SCSS variables renamed!

**Pull request merged**